src/pudl/settings.py

```diff
@@ -58,7 +59,7 @@ def validate_partitions(cls, partitions):  # noqa: N805
             if len(partitions_not_working) > 0:
                 raise ValueError(
                     f"'{partitions_not_working}' {name} are not available.")
-            partitions[name] = sorted(set(partition))
+            partitions[name] = tuple(sorted(set(partition)))
```

Comment:
IIRC the `tuple()` was just to ensure immutability of the parameters in `WORKING_PARTITIONS`, since they're used as defaults for a bunch of functions and we didn't want them getting mutated. Since the Pydantic models are being defined as immutable I think just using the sorted list should be fine?

Reply:
Good point. I will remove `tuple()`.